docs/CHANGELOG.asciidoc

```diff
@@ -185,6 +184,9 @@ APM::
 * Adds table tabs showing summary of metrics {kibana-pull}153044[#153044]
 * Adds warning to Edit Rule Flyout when publicUrl is not configured {kibana-pull}149832[#149832]
 
+Cases::
+* Adds support for file attachments in Cases {kibana-pull}154436[#154436]
```

Comment:
Could you move the following into the Cases section from the Alerting section:

- Adds the Cases column to the alerts table [#150963](https://github.com/elastic/kibana/pull/150963)
- Adds filtering and sorting for the case activity [#149396](https://github.com/elastic/kibana/pull/149396)
- Adds the ability to filter user activities with pagination [#152702](https://github.com/elastic/kibana/pull/152702)